src/transport/AdminPairingTable.h

```diff
@@ -194,8 +194,7 @@ class DLL_EXPORT AdminPairingInfo
 
         Crypto::P256SerializedKeypair mOperationalKey;
         uint8_t mRootCert[kMaxChipCertSize];
-        uint8_t mOperationalCert[kMaxChipCertSize];
-
+        uint8_t mOperationalCert[kMaxChipCertSize * 2];
```

Comment:
Why was this needed?

Reply:
We are storing an array of certs now, NOC and ICA certs. The CPP file has the comment that reflects it.

Reply:
I'll add the comment here as well.